.github/workflows/lint.yaml

```
@@ -0,0 +1,17 @@
+name: Run Lint
+on: 
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '16'
+    - run: npm install

```

Maybe use `npm ci` instead of `npm install` https://www.geeksforgeeks.org/difference-between-npm-i-and-npm-ci-in-node-js/
```suggestion
    - run: npm ci
```

---

I didn't wanna use `ci` because someone might have a different package-lock.json (updated deps) which would throw. And we only care about the "linting part" here.